website/docs/docs/dbt-cloud-apis/discovery-use-cases-and-examples.md

```diff
@@ -874,35 +912,35 @@ Graph example:
 
 <VersionBlock firstVersion="1.6">
 
-### Which metrics are available? 
+### Which metrics are available?
 
 Metric definitions are coming soon to the Discovery API with dbt v1.6. You’ll be able to query metrics using the dbt Semantic Layer, use them for documentation purposes (like for a data catalog), and calculate aggregations (like in a BI tool that doesn’t query the SL).
```

Comment:
with the release of v1.6, isn't "Metric definitions are coming soon to the Discovery API with dbt v1.6." now stale?